### PR TITLE
Feature/track artists ref

### PR DESCRIPTION
**Pull request by radeklos**

### CODE REVIEW

**Comment by radeklos** [comment]
@krak3n @peeklondon CR please.

File: migrations/versions/f5babb123eb_.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ### end Alembic commands ###
+
+
+def downgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index(op.f('ix_track_artist_updated'), table_name='track_artist')
+    op.drop_index(op.f('ix_track_artist_track_id'), table_name='track_artist')
+    op.drop_index(op.f('ix_track_artist_created'), table_name='track_artist')
+    op.drop_index(op.f('ix_track_artist_artist_id'), table_name='track_artist')
+    op.drop_table('track_artist')
```
Comment (krak3n) [inline migrations/versions/f5babb123eb_.py]: Before we do any table dropping maybe we could have a little migration script that goes over our current list of album artists and tracks and fills this table?

Or could call the Spotify API to get the track artists on it's own.

Just don't wanna loose any data?

Comment (radeklos) [inline migrations/versions/f5babb123eb_.py]: It's a downgrade migration. I would like to populate this table with separated script which doesn't need to be merged.

Comment (krak3n) [inline migrations/versions/f5babb123eb_.py]: Oh yes ofc, ignore me :+1:

File: tests/test_auth.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def test_returns_resource(self, _is_authenticated_request):
 class TestIsAuthenticatedRequest(object):
 
     def test_returns_false_by_default(self):
-        assert is_authenticated_request() == False
+        assert is_authenticated_request() is False
```
Comment (krak3n) [inline tests/test_auth.py]: `assert not is_authenticated_request()` also works, dunno whats more `pythonic`

**Comment by krak3n** [comment]
:+1:

File: tests/test_auth.py (hunk 2 above)
Comment (radeklos) [inline tests/test_auth.py]: I know but it doesn't change the type. 
- not 'a' -> False
- not 5 - > False
- ...

Comment (krak3n) [inline tests/test_auth.py]: :+1:

**Pull request merged**